handlers/main.yml

```diff
@@ -15,6 +15,9 @@
 - name: postmap generic
   command: postmap hash:/etc/postfix/generic
 
+- name: postmap header_checks
+  command: postmap hash:/etc/postfix/header_checks
```

Comment:
Shouldn't this be:

```
command: postmap {{ postifx_header_check_format }}:/etc/postfix/header_checks
```

?

Reply:
yep you are right, but this gets changed to "postfix_default_database_type" in the other pull request which includes this one as well.